presto-server-rpm/src/main/resources/dist/etc/rc.d/init.d/presto

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Manages a Presto node service
+#
+# chkconfig: 345 85 15
+# description: Presto node
+#
+### BEGIN INIT INFO
+# Provides:          presto
+# Short-Description: presto node
+# Default-Start:     3 4 5
+# Default-Stop:      0 1 2 6
+# Required-Start:    $syslog $remote_fs
+# Required-Stop:     $syslog $remote_fs
+# Should-Start:
+# Should-Stop:
+### END INIT INFO
+
+SCRIPT_NAME=$0
+ACTION_NAME=$1
+SERVICE_NAME='presto'
+
+log_uid=$(date "+%Y.%d.%m-%H.%M.%S")
+
+# Launcher Config. 
+# Use data-dir from node.properties file (assumes it to be at /etc/presto). For other args use defaults from rpm install
+NODE_PROPERTIES=/etc/presto/node.properties
+DATA_DIR=$(grep -Po "(?<=^node.data-dir=).*" $NODE_PROPERTIES)
+CONFIGURATION="--launcher-config /usr/lib/presto/bin/launcher.properties --data-dir $DATA_DIR --node-config $NODE_PROPERTIES --jvm-config /etc/presto/jvm.config --config /etc/presto/config.properties --launcher-log-file /var/log/presto/launcher_$log_uid.log --server-log-file /var/log/presto/server_$log_uid.log"
+
+source /etc/presto/env.sh
+
+start () {
+    echo "Starting ${SERVICE_NAME} "
+    if [ -z "$JAVA8_HOME" ]
+    then
+        echo "Warning: No value found for JAVA8_HOME. Default Java will be used."
+        /usr/lib/presto/bin/launcher start ${CONFIGURATION}
+    else
+        PATH=${JAVA8_HOME}/bin:$PATH /usr/lib/presto/bin/launcher start ${CONFIGURATION}
+    fi
+    return $?
+}
+
+stop () {
+    echo "Stopping ${SERVICE_NAME} "
+    /usr/lib/presto/bin/launcher stop ${CONFIGURATION}
```

Comment:
In my testing, stop and status did not work with no CONFIGURATION passed. status would return wrong status and stop wont actually stop the server running. Looks like launcher assumes default config values otherwise.